src/System.Private.CoreLib/src/System/Globalization/DateTimeFormatInfo.cs

```diff
@@ -79,19 +81,19 @@ public sealed class DateTimeFormatInfo : IFormatProvider, ICloneable
         // Caches for various properties.
         //
 
-        private String _amDesignator = null;
```

Comment:
I will change the names to match the guidelines in another PR. This version is a wholesale copy from coreclr.